### DISCUSSION TITLE
IOException Error Number: 2 on Windows

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#22] [NCIRF] [answered]
We recently identified the cause of the error, "IOException Error Number:2" which has been reported by some users in the past. We found that the error is caused when Windows username has a space(s), for example "John Doe" In this case, System Environment Variables are not properly defined for Geant4. When we removed the space in username, NCIRF runs fine.

**Comment by mathomas1481** [comment]
Choonsik: I just installed the new v3.0 of NCIRF. I am getting the same error as described above, even when I run the default batch input files you've provided. I have been able to run all previous versions of NCIRF without issue. I don't think my Windows user has a space like what you refer to above. Any ideas why I'm having trouble only with NCIRF v3.0?

**Reply by choonsiklee** [reply, marked as answer]
Apologies for the inconvenience. We've identified some issues with the packaging of the Windows version, where the Geant4 and phantom libraries were missing. Please download NCIRF3.0.20241217 and try again.

**Comment by mathomas1481** [comment]
This new version works great. Thanks, Choonsik!
